jwst/flatfield/tests/test_flatfield.py

```diff
@@ -265,13 +265,23 @@ def test_nirspec_fs_flat(srctype):
     data.slits[0].xsize = shape[1]
     data.slits[0].ysize = shape[0]
 
-    flats = create_nirspec_flats(w_shape)
+    orig_data = data.copy()
+    flats = create_nirspec_flats(w_shape, flat_data_value=0.9)
     result = FlatFieldStep.call(data, override_fflat=flats[0], override_sflat=flats[1],
                                 override_dflat=flats[2], override_flat='N/A')
 
-    # null flat, so data is the same, other than nan edges
     nn = ~np.isnan(result.slits[0].data)
-    assert np.allclose(result.slits[0].data[nn], data.slits[0].data[nn])
+    assert_allclose(result.slits[0].data[nn], 1.371742, rtol=1e-6)  # FIXME: should be orig_data.slits[0].data[nn] / 0.9 ?
```

Comment:
I am stuck designing a test for the inverse roundtrip. Would need a flatfield step expert to help here.

Reply:
https://innerspace.stsci.edu/display/SCSB/JWST+calibration+pipeline+maintainers says @jemorrison @stscirij @melanieclarke ? 🙏

Reply:
Got it. Thanks!